lib/orcid.js

```diff
@@ -67,6 +71,63 @@ function getEmailForPerson(orcid, accessToken, callback) {
   });
 }
 
+function getPersonEducations(orcid, callback) {
```

Comment:
I got stumped here by the method name as it just reads funny - I was thinking maybe we can do "getPersonEducationData" or sth.
Then I saw the URL in the ORCID API so let's follow what their convention here :D